extensions/notebook/src/book/bookModel.ts

```diff
@@ -37,6 +37,8 @@ export class BookModel implements azdata.nb.NavigationProvider {
 	}
 
 	public async initializeContents(): Promise<void> {
+		this._tableOfContentPaths = [];
```

Comment:
out of curiosity what change necessitates initializing these variables?

Reply:
If am not resetting them here, when I initialize for refresh on toc changes, there are two entries in them and that's causing the tree to have the old values.

---
In reply to: [388555362](https://github.com/microsoft/azuredatastudio/pull/9453#discussion_r388555362) [](ancestors = 388555362)